wp-content/plugins/core/src/Assets/Admin/Styles.php

```diff
@@ -43,4 +43,28 @@ public function enqueue_login_styles() {
 		wp_enqueue_style( 'tribe-styles-login' );
 	}
 
+	/**
+	 * This function removes the `wp-reset-editor-styles` style dependency from the
+	 * block editor styles enqueue.
+	 *
+	 * The Block Editor in WP Core v5.8.0 is enqueuing a reset.css file which overrides
+	 * many of our baseline typographic styles for paragraphs, headings, etc. due to the
+	 * specificity applied inside the `.editor-styles-wrapper` class.
+	 *
+	 * This function may well be able to removed when the block editor is fully iframed
+	 * per the Gutenberg issue discussion linked below.
+	 *
+	 * See https://github.com/WordPress/gutenberg/pull/33522 for context.
+	 *
+	 * @action admin_enqueue_scripts
+	 */
+	public function remove_editor_style_reset(): void {
+		if ( ! isset( wp_styles()->registered['wp-edit-blocks'] ) ) {
```

Comment:
```suggestion
		if ( ! isset( wp_styles()->registered['wp-edit-blocks']->deps ) ) {
```

Would feel much more comfortable checking for the exact property in the edge case WordPress has `registered['wp-edit-blocks']` set but not `registered['wp-edit-blocks']->deps`